### PR TITLE
Ipfs Pinata Tests

### PR DESCRIPTION
**Pull request by ricklove**
- tests for ipfs-pinata plugin

### NOTE: These tests require an .env variable to be provided with a pinata api keys.

- For CICD tests, these ENV VARS should be added to secrets:

```
pinataJwtToken=eyJ...
```

### CODE REVIEW

**Review by sinapsist — APPROVED** [review]
LGTM

**Review by jchenche — APPROVED** [review]

**Comment by GImbrailo** [comment]
@hu3man @alexzbusko here is the PR I was talking about earlier.

**Comment by hu3man** [comment]
Oops looks like I did delete the branch by accident 🤦‍♂️

Opening back up and assigning to @alexzbusko to drive

File: .github/workflows/main.yml [hunk 1]
```diff
@@ -4,6 +4,7 @@ on:
   pull_request:
     branches:
       - main
+      - 893-ipfs-pinata-pin-task
```
Comment (GImbrailo) [inline .github/workflows/main.yml:7]: Should this line be removed before merging into `main`?

**Review by GImbrailo — CHANGES_REQUESTED** [review]

**Pull request merged**